### ISSUE TITLE
egt sensor

### ISSUE THREAD

**Issue by DEN3535** [Switchleg1/SimosTools#8]
Hello.  How can I create an additional exhaust gas temperature "sensor" and write it to log or change an existing but not important "sensor" to an egt sensor?  What settings should be?

**Comment by Switchleg1** [comment]
You would need to have a custom ASW patch for your ECU to allow for this.  This is something I have added to later versions of Switchpatch that is not being released to the general public.

**Comment by DEN3535** [comment]
Under what conditions can I get this patch?

**Comment by Switchleg1** [comment]
first off who is tuning your car?  are you currently using any version of my patch (Switchpatch)?

**Comment by DEN3535** [comment]
No, I haven't used the patch yet.  Etuners st1 flashed but not simos tools.  I'm just looking into using simos tools for flashing.  I use it as a logger only.
